wcfsetup/install/files/lib/data/user/activity/event/ViewableUserActivityEventList.class.php

```diff
@@ -94,7 +94,7 @@ public function readObjects()
     }
 
     /**
-     * Returns timestamp of oldest entry fetched.
+     * Returns timestamp of the oldest entry fetched.
```

Comment:
Not incorrect, but incomplete. It should be “Returns the timestamp”.

Reply:
Whoops.